axoproject/src/javascript.rs

```diff
@@ -1,5 +1,16 @@
 //! Support for npm-based JavaScript projects
 
+/// The extension to add when invoking JS programs which aren't proper executables.
+///
+/// This is necessary for invoking e.g. `npm` on Windows, where it's actually `npm.cmd`
+/// when invoked via `std::process::Command` (as opposed to typed in a shell).
+///
+/// See: <https://doc.rust-lang.org/nightly/std/process/struct.Command.html#platform-specific-behavior>
+#[cfg(windows)]
+pub const JS_PROGRAM_EXT: &str = ".cmd";
+#[cfg(not(windows))]
+pub const JS_PROGRAM_EXT: &str = "";
```

Comment:
Unfortunately `cfg` didn't pick up on this being one docstring for either definition.